scaffold/template/custom/webpack.config.js

```diff
@@ -35,103 +35,117 @@ const babelOptions = require("./babel.config.js")
 const appDirectory = path.resolve(__dirname)
 
 const isProduction =
-  process.argv[process.argv.indexOf("--mode") + 1] === "production"
+    process.argv[process.argv.indexOf("--mode") + 1] === "production"
 
 const html_template_path = isProduction
-  ? "./public/django_index.html"
-  : "./public/index.html"
+    ? "./public/django_index.html"
+    : "./public/index.html"
 
 const HTMLWebpackPluginConfig = new HTMLWebpackPlugin({
-  template: path.resolve(appDirectory, html_template_path),
-  filename: "index.html",
-  inject: false
+    template: path.resolve(appDirectory, html_template_path),
+    filename: "index.html",
+    inject: false
 })
 
 const DevEnvPlugin = new webpack.DefinePlugin({
-  __DEV__: process.env.NODE_ENV !== "production" || true,
-  "process.env.NODE_ENV": JSON.stringify(process.env.NODE_ENV || "development")
+    __DEV__: process.env.NODE_ENV !== "production" || true,
+    "process.env.NODE_ENV": JSON.stringify(process.env.NODE_ENV || "development")
 })
 
-const JestWorkerPlugin = new webpack.EnvironmentPlugin({ JEST_WORKER_ID: null })
+const JestWorkerPlugin = new webpack.EnvironmentPlugin({JEST_WORKER_ID: null})
 
 // This is needed for webpack to compile JavaScript.
 // Many OSS React Native packages are not compiled to ES5 before being
 // published. If you depend on uncompiled packages they may cause webpack build
 // errors. To fix this webpack can be configured to compile to the necessary
 // `node_module`.
 const babelLoaderConfiguration = {
-  test: /\.js$/,
-  // Add every directory that needs to be compiled by Babel during the build.
-  include: [
-    path.resolve(appDirectory, "index.js"),
-    path.resolve(appDirectory, "App.js"),
-    path.resolve(appDirectory, "modules"),
-    path.resolve(appDirectory, "screens"),
-    path.resolve(appDirectory, "options"),
-    path.resolve(appDirectory, "store"),
-    path.resolve(appDirectory, "node_modules/react-native-reanimated")
-  ],
-  use: {
-    loader: "babel-loader",
-    options: babelOptions
-  }
+    test: /\.js$/,
+    // Add every directory that needs to be compiled by Babel during the build.
+    include: [
+        path.resolve(appDirectory, "index.js"),
+        path.resolve(appDirectory, "App.js"),
+        path.resolve(appDirectory, "modules"),
+        path.resolve(appDirectory, "screens"),
+        path.resolve(appDirectory, "options"),
```

Comment:
Looks like this file is adding 4-space identation. I don't think this is correct for this repo?